Docs/source/running_cpp/parameters.rst

```diff
@@ -355,6 +355,10 @@ Particle initialization
     particles are pushed in a standard way, using the specified pusher.
     (see the parameter ``<species_name>.zinject_plane`` below)
 
+* ``particles.do_tiling`` (`bool`) optional (default `false`)
```

Comment:
```suggestion
* ``particles.do_tiling`` (`bool`) optional (default `true` if WarpX is compiled for GPUs, `false` otherwise)
```

Reply:
I would suggest to add that if WarpX is compiled for GPUs the default value of `do_tiling` is `false`